src/cf.data.pre

```diff
@@ -2868,10 +2900,6 @@ DOC_START
 	  tcp_outgoing_address 2001:db8::1
 	  tcp_outgoing_address 10.1.0.3
 
-	Processing proceeds in the order specified, and stops at first fully
-	matching line.
-
-	Squid will add an implicit IP version test to each line.
 	Requests going to IPv4 websites will use the outgoing 10.1.0.* addresses.
 	Requests going to IPv6 websites will use the outgoing 2001:db8:* addresses.
```

Comment:
These two (old) lines belong to the (old) example above them. I moved the two non-example statements ("Processing proceeds..." and "Squid will add...") up, so that they do not split this long example. That split caused problems in earlier branch commits (that accidentally inserted even more text in the middle of that old example), and I bet similar problems may happen in the future.